content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -146,107 +146,18 @@ $ kubectl apply -f <(istioctl kube-inject \
      --filename $ORIGINAL_DEPLOYMENT_YAML)
 {{< /text >}}
 
-## Migrating per-service mutual TLS enablement via annotations to authentication policy
+### Migrating from `RbacConfig` to `ClusterRbacConfig`
```

Comment:
Why do we need this in 1.2  I recall something related to this in 1.1. https://istio.io/docs/setup/kubernetes/upgrade/steps/#migrating-from-rbacconfig-to-clusterrbacconfig

Reply:
at the minimum, we can point to 1.1's note on this vs duplicating things.

Reply:
Removed.